### PR TITLE
Remove unstable rustfmt-nightly features

### PR DESCRIPTION
**Pull request by nox**

### CODE REVIEW

File: tests/rustfmt.toml [hunk 1]
```diff
@@ -1 +1 @@
-error_on_line_overflow_comments = false
+# Untentionally left blank.
```
Comment (emilio) [inline tests/rustfmt.toml]: Untentionally? :)

File: rustfmt.toml [hunk 2]
```diff
@@ -1,10 +1,6 @@
 max_width = 80
-format_strings = false
 fn_brace_style = "SameLineWhere"
 item_brace_style = "SameLineWhere"
 struct_lit_multiline_style = "ForceMulti"
 where_trailing_comma = true
-reorder_imports = true
-reorder_imported_names = true
 normalize_comments = false
-write_mode = "Overwrite"
```
Comment (emilio) [inline rustfmt.toml]: Huh, I'm really surprised this is unstable now :(

Comment (nox) [reply rustfmt.toml]: Cc @nrc

**Pull request merged**